src/BenchmarksApps/HttpClient/Proxy/ProxyExtensions.cs

```diff
@@ -126,6 +127,14 @@ public static async Task CopyProxyHttpResponse(this HttpContext context, HttpRes
             }
         }
 
+        public static HttpResponseMessage CreateResponse(HttpRequestMessage request, byte[] responseBody)
+        {
+            var response = new HttpResponseMessage(System.Net.HttpStatusCode.OK);
+            CopyHeaders(response.Headers, request.Headers);
+            response.Content = new ByteArrayContent(responseBody);
```

Comment:
You still need to set the ContentLength.

Reply:
Done, but I don't see any impact on RPS.

Reply:
I discovered that it does increase RPS but only in one scenario - 1024B.